app/graphql/types/customers/object.rb

```diff
@@ -51,7 +51,7 @@ class Object < Types::BaseObject
       field :deleted_at, GraphQL::Types::ISO8601DateTime, null: true
       field :updated_at, GraphQL::Types::ISO8601DateTime, null: false
 
-      field :active_subscription_count, Integer, null: false, description: 'Number of active subscriptions per customer'
+      field :active_subscriptions_count, Integer, null: false, description: 'Number of active subscriptions per customer'
```

Comment:
Layout/LineLength: Line is too long. [121/120] (https://rubystyle.guide#max-line-length)